src/tools/genpolicy/update_policy_samples.py

```diff
@@ -1,3 +1,4 @@
+from concurrent.futures import ThreadPoolExecutor
```

Comment:
Please assess whether parallel execution may have an effect on our layer cache.

Reply:
Agreed - this is a good test case to detect possible cache corruptions. But:
- It might cost us some time, and
- If the cache gets corrupted, it might be hard to debug the corruption due to the random nature of parallel execution

So, I don't have a strong preference between parallel or serialized execution here.